.github/actions/environment/action.yml

```diff
@@ -21,16 +21,15 @@ runs:
         # run from it. So running it only outside of docker container.
         # Also on local hosts, like that we use for 'freebsd' workflow
         # testing, 'sudo' not acceptable outside of 'freebsd' virtual
-        # machine and to avoid of hangs let's check sudo run ability
-        # with command like 'timeout 2 sudo ls /sbin/swapoff'.
+        # machine and to avoid of hangs let's run it with '-n' flag.
         # NOTE: To switch off swap from inside github 'container' tag
         # additional memory flags should be added to its 'options' tag:
         #   options: '--memory=<some value, like 7G> --memory-swap=<the same value as for memory option>'
         if which free ; then
             echo "Check initial swap memory values:"
             free
         fi
-        if timeout 2 sudo ls /sbin/swapoff ; then
+        if sudo -n ls /sbin/swapoff ; then
```

Comment:
If we don't have permissions for an operation, what do we expect as a result of the script? Common `sudo` failure or our own explanation?

Reply:
Right, in this way common 'sudo' failure message would be completely enough to see the cause of the issue.